mod-pubsub-server/src/main/java/org/folio/dao/impl/EventDescriptorDaoImpl.java

```diff
@@ -118,16 +118,9 @@ private String getQueryByEventTypes(List<String> eventTypes) {
   }
 
   @Override
-  public Future<Boolean> delete(String eventType) {
-    Promise<RowSet<Row>> promise = Promise.promise();
-    try {
+  public Future<Void> delete(String eventType) {
       String query = format(DELETE_BY_ID_SQL, MODULE_SCHEMA, TABLE_NAME);
-      pgClientFactory.getInstance().execute(query, Tuple.of(eventType), promise);
-    } catch (Exception e) {
-      LOGGER.error("Error deleting EventDescriptor with event type '{}'", eventType, e);
-      promise.fail(e);
-    }
-    return promise.future().map(updateResult -> updateResult.rowCount() == 1);
+      return pgClientFactory.getInstance().execute(query, Tuple.of(eventType)).mapEmpty();
```

Comment:
In PubSubImp class where main invoking point of the service also catches the same exception and also handles with otherwise(), Do you want me to add the logging with try catch any way?